lib/renderer/web-view/web-view-impl.ts

```diff
@@ -249,6 +249,25 @@ export const setupMethods = (WebViewElement: typeof ElectronInternal.WebViewElem
   WebViewElement.prototype.capturePage = async function (...args) {
     return deserialize(await ipcRendererInternal.invoke('ELECTRON_GUEST_VIEW_MANAGER_CAPTURE_PAGE', this.getWebContentsId(), args))
   }
+
+  const createPropertyHandler = function (method: string, type: 'SET' | 'GET') {
```

Comment:
The enum here seems unnecessary, can we have `createPropertyGetterHandler` and `createPropertySetterHandler` as two separate things.

Reply:
i'm impartial, this was just slightly less code but if you'd prefer that i'll get it updated :)

Reply:
done!